sdk/quantum/quantum-jobs/generate.ps1

```diff
@@ -0,0 +1,2 @@
+autorest  --version=3.0.6335 --typescript --use=@autorest/modelerfour@4.15.452 swagger\README.md
```

Comment:
This file can be removed. Normally we do not include this kind of ps files in our repo.